app/views/shared/_npc_show.html.erb

```diff
@@ -130,10 +130,10 @@
       <h3>Spells:</h3>
         <p>Spell_slots: <%= @npc.spell_slots %></p>
         <p>Current Spells:</p>
-          <% if @spells.class == String %>
-            <%= @spells %>
+          <% if @npc.spells.class == String %>
+            <%= @npc.spells %>
           <% else %>
-            <% @spells.flatten.each do |spell| %>
+            <% @npc.spells.each do |spell| %>
               <li>Name: <%= spell[:name] %></li>
               <li>Description: <%= spell[:description] %></li>
```

Comment:
All updated d/t having access to spells, weapons, packs, and armor via model relations